README

```diff
@@ -46,6 +46,18 @@ choice is used when the daemon is restarted (e.g. with the machine reboot).
 To disable all tunings, run:
 # tuned-adm off
 
+To show information/description of given profile or current profile if no profile is specified, run:
+# tuned-adm profile_info
+
+To verifies current profile against system settings, run:
```

Comment:
I think this should spell either:
'To verify....' or 'Verifies...', but if there is ', run:', I think it should be 'To verify:'